department-of-veterans-affairs/va.gov-cms
# VACMS-3924: Add benefit limit validation to clp.

## Description
See #3924 

## Testing done
Manual / Behat

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/104188543-64e1c300-53e7-11eb-8e16-ce5f8290ba74.png)

## QA steps
- As an administrator, go to node/1505/edit
- Try to save node, and visually verify that the following error appears `Please select no more than two VA Benefits`
- Deselect one of the benefits, save, and visually verify that errors related to benefit limits are not thrown.

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
